overcloud.yml

```diff
@@ -51,6 +51,13 @@
         path: /home/stack/templates
       ignore_errors: true
 
+    - name: workaround for BZ 2123252
+      become: true
+      shell: |
+        dnf install containernetworking-plugins -y
+        sed -i 's/netavark/cni/g' /etc/containers/containers.conf
+      when: osp_release|int == 17
```

Comment:
use >= 17

Reply:
we are hitting this bug on osp 17, so i have added workaround for now till the fix is released